### DISCUSSION TITLE
S3 New Form Upload Field - Cannot Connect my S3 Source

### DISCUSSION THREAD

**Discussion by superbolt-x** [Budibase/budibase#4515] [Help]
Hi!

I connected and tested my S3 source using s3.us-east-1.amazonaws.com as endpoint and worked. But then not sure when I use an S3 upload form field component how to select that source, the option "S3 Datasource" is empty. Tried also with a DP with the S3 query but nothing

How should I proceed? [Self Hosted on AWS EC2 - BB v1.0.62]

Thanks so much!

**Comment by efefirda** [comment]
same issue here. do you solved it? @superbolt-x

**Reply by superbolt-x** [reply]
Nope, no luck so far - I even upgraded to 1.0.73 but still nothing

**Reply by mjashanks** [reply]
@superbolt-x @efefirda  - we're looking into this. Looks like a bug

**Comment by ajay-nagarajan** [comment]
Hey @mjashanks! could you please give us an indication as to how long this might take to be addressed? 
we are using 1.0.82 of budibase and it doesn't seem to be fixed in that version either

**Reply by mjashanks** [reply]
Hi @ajay-nagarajan - the issue is logged here: https://github.com/Budibase/budibase/issues/4814

It's not been fixed in the most recent release either, but I will try and get it prioritised for this month.

**Reply by mjashanks** [reply]
Update - this has actually been fixed in the latest release: `1.0.86`

**Reply by ajay-nagarajan** [reply]
Thanks @mjashanks ! 👍

**Reply by sgyotk** [reply]
Hey @mjashanks , I have the same problem - still in 1.0.90, no S3 Datasource to chose from in the S3-Uploader, connecting and quering in the Data Source works fine though, S3 Datasource stays empty, as initially described. Thanks!
